ros/src/twist_controller/twist_controller.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, cp):
             max_steer_angle=cp.max_steer_angle)
 
         self.cp = cp
-        self.pid = PID(kp=5, ki=0.5, kd=0.5, mn=cp.decel_limit, mx=cp.accel_limit)
+        self.pid = PID(kp=0.3, ki=0.1, kd=0.0, mn=cp.decel_limit, mx=cp.accel_limit)
```

Comment:
I am ok with using the previous values.
Also, I suggest using the default parameters for the low pass filter (below two lines), the reasons are
1. No significant performance gain of using tau=3, ts=1.
2. the value tau=0.5, ts=0.2 may be critical for the actual hardware on Carla

Reply:
you are right with using the default values. This will be more stable on different environments